blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -83,20 +129,89 @@ const MediumPromoItem = ({ customFields, arcSite }) => {
     }`,
 		}) || null;
 
-	return <MediumPromoPresentation content={content} {...customFields} />;
-};
-
-const MediumPromo = ({ customFields }) => {
-	const { isAdmin, arcSite } = useFusionContext();
-	const shouldLazyLoad = customFields?.lazyLoad && !isAdmin;
 	if (shouldLazyLoad && isServerSide()) {
 		return null;
 	}
-	return (
+
+	const hasAuthors = showByline ? content?.credits?.by : null;
+	const contentDescription = showDescription ? content?.description?.basic : null;
+	const contentHeading = showHeadline ? content?.headlines?.basic : null;
+	const contentUrl = content?.websites?.[arcSite]?.website_url;
+	const imageSearchField = imageOverrideURL ? "imageOverrideURL" : "imageURL";
+	const promoImageURL = imageOverrideURL || getImageFromANS(content);
+	const contentDate = content?.display_date;
+	const formattedDate = Date.parse(contentDate)
+		? localizeDateTime(new Date(contentDate), dateTimeFormat, language, timeZone)
+		: "";
+
+	const MediaImage = () =>
+		showImage ? (
+			<Conditional component={Link} condition={contentUrl} href={contentUrl}>
+				<Image
+					alt={content?.headlines?.basic}
+					src={promoImageURL || fallbackImage}
+					searchableField
+					data-aspect-ratio={imageRatio?.replace(":", "/")}
+				/>
+			</Conditional>
+		) : null;
+
+	return showHeadline || showImage || showImage || showDescription || showByline || showDate ? (
```

Comment:
```suggestion
	return showHeadline || showImage || showDescription || showByline || showDate ? (
```
For LGTM message

Reply:
Github wouldn't let me update here saying the change was outdated but I did make the change.